src/ui/inspector/Location.vue

```diff
@@ -44,45 +46,31 @@ export default {
         this.openmct.selection.off('change', this.updateSelection);
     },
     methods: {
-        getObjectPath(domainObject) {
-            let objectPath = [],
-                context = domainObject.getCapability('context');
-
-            if (context) {
-                objectPath = context.getPath();
-            }
-
-            return objectPath;
-        },
-        itemDecorator(d) {
-            let domainObject = d.useCapability('adapter'),
-                key = this.openmct.objects.makeKeyString(domainObject.identifier),
-                childObjectPath = this.getObjectPath(d)
-                    .slice(1)
-                    .reverse()
-                    .map((dd) => dd.useCapability('adapter'));
-
-            return {
-                domainObject,
-                key,
-                objectPath: childObjectPath 
-            }
+        makeParentObjects() {
+            this.originalPath.forEach((pathObject) => {
+                this.openmct.objects.getOriginalPath(pathObject.key)
+                    .then(path => {
+                        this.$set(
+                            this.parentObjects,
+                            pathObject.key,
+                            {
+                                domainObject: pathObject.domainObject,
+                                key: pathObject.key,
+                                objectPath: path.slice(1).reverse()
```

Comment:
See my comment on `getOriginalPath`, but I think we should use the same ordering we use for object paths elsewhere.